### PR TITLE
61 never open guidance

### LINKED ISSUE #61: Never open guidance
Requires a clear example, guidance and some considerations. RL to follow up with fintrans & LE regarding this bit.

### PR DESCRIPTION
**Pull request by r-leyshon**
## Description/Contribution Summary

Fixes #61 

## Motivation and Context

Guidance has now incorporated feedback from ONS FoI team & DSC colleagues in Email chain.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Accessibility

No errors, one known alert to tolerate - A link to a PDF document is present.

## Checklist:

- [X] I have read the [CONTRIBUTING.md](../../CONTRIBUTING.md) guidance for raising pull requests  
- [X] I have followed the intended structure of this project
- [X] I have performed a self-review
- [X] This pull-request meets the accessibility guidance provided in the contributing guidance.

### CODE REVIEW

**Comment by NicciPotts** [comment]
> @NicciPotts @ethan-moss @TSGreen Hi all, I just resolved a small merge conflict in the qmd YAML. Does anyone have a couple of hours to review this? Would need draft checking for typos etc and some more opinionated feedback on language used and points made would be welcome, but not required.

Hi @r-leyshon I've got a really short week this week (this afternoon and tomorrow) but if I get my model running tomorrow I should be able to give this a look. *IF*

File: never-open/explanation.qmd [hunk 1]
```diff
@@ -1,151 +1,266 @@
...
+Some of the main risks that closed source development may introduce are as
+follows:
+
+#### Introducing poor practice
+
+Relaxing the controls around quality assurance of code may result in bugs or
+errors going undiscovered. Ensure that the development team understands the
+good practice principles set out in the ONS duck book @DuckBook. Ensure that
+peer review of code is practised regularly, ideally at the point of PR to the
+main branch. Checks and balances should be proportionate to the risk involved
```
Comment (TSGreen) [inline never-open/explanation.qmd:118]: The two sentences that start with "ensure" don't really fit into narrative of the wider paragraph

**Review by TSGreen — CHANGES_REQUESTED** [review]
Hi Rich, I had a read and made a bunch of small edits (largely in relation to formatting for rendering), which I will push up shortly and I've added a couple of specific comments for things I wasn't sure about.

File: never-open/explanation.qmd [hunk 2]
```diff
@@ -1,151 +1,266 @@
...
-significant resource applied to quality assurance of the codebase and any
-version history to be released. Any delay to responding to the request could
-result in an unlawful breach of the Freedom of Information Act 2000@FOIAct. 
-
-### Not all sensitive projects should completely close source
+In this article, the use case of closed-source has been introduced.
+Misconceptions about closed source development and risk avoidance have been
+asserted. Key risks about code quality and the potential for FoI requests have
+been discussed at length and ONS FoI team have answered questions in regards to
+the risk and scope of potential FoI requests. This article intends to help the
```
Comment (TSGreen) [inline never-open/explanation.qmd:262]: It in unclear in the current version of the page that the FoI team were asked questions and which part of the page this refers to. So might be worth updating this.

**Comment by r-leyshon** [comment]
Some good catches in that latest commit @TSGreen . I don't know what's going on with the referencing syntax, but I find sometimes it accepts `some text@ref_with_no_space` and on other occasions trying to do that completely breaks all references in the document. I can see you must have encountered this problem when locally renderring. Possibly worth raiseing a bug report on the quarto repo if only I could reproduce the difference.

File: never-open/explanation.qmd (hunk 2 above)
Comment (r-leyshon) [reply never-open/explanation.qmd]: Thanks again @TSGreen , I've now implemented changes in response to your comments. Please review and merge if appropriate.

**Review by TSGreen — APPROVED** [review]
Looks good! Will merge

**Pull request merged**